contracts/Migrator.sol

```diff
@@ -127,20 +127,21 @@ contract Migrator is Governable {
         uint256 newStakeAmount,
         uint256 newStakeDuration
     ) external isSolvent {
-        if (lockupIds.length == 0) {
-            revert LockupIdsRequired();
-        }
-
-        // Unstake
-        (uint256 ogvAmountUnlocked, uint256 rewardsCollected) = ogvStaking.unstakeFrom(msg.sender, lockupIds);
-
-        if (migrateRewards) {
-            // Include rewards if needed
+        if (lockupIds.length > 0) {
+            // Unstake if there are any lockups
+            (uint256 ogvAmountUnlocked, uint256 rewardsCollected) = ogvStaking.unstakeFrom(msg.sender, lockupIds);
+
+            if (migrateRewards) {
+                // Include rewards if needed
+                ogvAmountFromWallet += rewardsCollected;
```

Comment:
Is the approval on the frontend adding extra?

Reply:
Yep. It already happens when someone tries to unstake lockups with rewards. This changeset just enables users to claim rewards and migrate them without unstaking any lockups